### ISSUE TITLE
false positive "no-unused-import": type used in using...for directive

### ISSUE THREAD

**Issue by 0xCourtney** [solhint-community/solhint-community#29]
For `SolhintTest.sol`, solhint reports that `EnumerableSet` is not used.

```solidity
// EnumerableSetUD60x18.sol
pragma solidity >=0.8.19;

import {EnumerableSet} from "@solidstate/contracts/data/EnumerableSet.sol";

library EnumerableSetUD60x18 {
    using EnumerableSet for EnumerableSet.Bytes32Set;
}
```

```solidity
// SolhintTest.sol
pragma solidity >=0.8.19;

import {EnumerableSetUD60x18, EnumerableSet} from "./libraries/EnumerableSetUD60x18.sol";

contract SolhintTest {
    using EnumerableSetUD60x18 for EnumerableSet.Bytes32Set;
}
```
